### ISSUE TITLE
Package-specific log level not working with writing thread (SLF4J)

### ISSUE THREAD

**Issue by lenis0012** [tinylog-org/tinylog#77]
When writingthread is enabled, the package-specific logging level does not work as expected
With the following config, debug messages are not shown:
```
tinylog.writer=console
tinylog.level=info
tinylog.format={date:HH:mm:ss} {level} {class_name} - {message}
tinylog.level@my.package=debug
tinylog.writingthread=true
```
If I remove the writingthread property, the debug messages are shwon as expected.
I assume this is a bug, since the writingthread property does not mention disabling package-specific log levels.
I am using version 1.3.5 of tinylog slf4j-binding.

**Comment by pmwmedia** [comment]
I have tested it on my machine and I got debug messages.

tinylog.properties (I renamed my.package to my.example as `package` is a reserved keyword)
```
tinylog.writer=console
tinylog.level=info
tinylog.format={date:HH:mm:ss} {level} {class_name} - {message}
tinylog.level@my.example=debug
tinylog.writingthread=true
```

Application.java
```java
package my.example;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Application {
	
	private final static Logger logger = LoggerFactory.getLogger(Application.class);

	public static void main(String[] args) {
		logger.info("Hello!");
		logger.debug("Bye!");
	}

}
```

Output:
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/D:/Entwicklung/Java/tinylog-1.3/slf4j-binding/target/classes/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/D:/Entwicklung/Java/tinylog-1.3/example/slf4j-binding-1.3.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.TinylogLoggerFactory]
17:07:50 INFO Application - Hello!
17:07:50 DEBUG Application - Bye!
```
[My Eclipse Project](https://github.com/pmwmedia/tinylog/files/2172859/example.zip)

**Comment by lenis0012** [comment]
@pmwmedia After reading your response I did some more testing
And I realized that one specific INFO message was not getting displayed either.
I assume that the writingthread is watching the wrong thread.

So most of my INFO messages triggered before the main thread exited.
Changing the observe property of the writingthread resolved the issue.

I am sorry for falsely reporting an issue, and thank you very much for taking the time to verify the issue and respond in such a short period of time. I really appreciate it.
